app/editor/src/features/admin/ingests/configurations/AudioClip.tsx

```diff
@@ -22,15 +22,10 @@ export const AudioClip: React.FC = (props) => {
         options={TimeZones}
         value={timeZone}
       />
-      <FormikText
-        label="Format"
-        name="configuration.format"
-        tooltip="Format of the clip"
-        placeholder="mp3"
-      />
+      <FormikText label="Format" name="configuration.format" tooltip="Format of the clip" />
```

Comment:
Removed placeholder to reduce confusion when using different formats.  FFmpeg can determine the format by the output file name.